.vnurc

```diff
@@ -27,3 +27,5 @@ Bad value “presentation” for attribute “role” on element “svg”.
 Element “input” is missing required attribute “aria-checked”.
 # https://github.com/w3c/aria-practices/issues/3070
 Attribute “aria-actions” not allowed on element “button” at this point.
+# https://github.com/w3c/aria-practices/issues/3265
+Element “search” not allowed as child of element “div” in this context. (Suppressing further errors from this subtree.)
```

Comment:
```suggestion
Element “search” not allowed as child of element “div” in this context.*
```

@jongund using a wildcard with that filter works. Going off `(Suppressing further errors from this subtree.)`, additional messages follow which aren't displayed in the log.

cc @mcking65